### ISSUE TITLE
Refuses to set unsafe User-Agent

### ISSUE THREAD

**Issue by jpfiorilla** [Airtable/airtable.js#52]
Hello,

I'm attempting to use the Airtable library to send a newsletter signup email to the appropriate api route.

I'm using the example Airtable gives me as follows:
```
var Airtable = require('airtable');
var base = new Airtable({apiKey: 'MY_API_KEY'}).base('appgDopuvXbayMxX3');

base('Newsletter Signup').create({ email: 'asdf@d.com' }, function(err, record) {
    if (err) { console.error(err); return; }
    console.log(record.getId());
});
```

However, I get the browser error 'Refused to set unsafe header "User-Agent"', about which I don't know what to do.

The response I get includes `message: "Unknown field name: "email"`. This confuses me as well, as the form I'm trying to submit to has an email field.

Appreciate any help!

**Comment by alecglassford** [comment]
What browser are you trying this in? I expect that updating it might solve the issue: airtable.js [sets a user agent header in its requests](https://github.com/Airtable/airtable.js/blob/master/lib/run_action.js#L21). It looks like this used to be a forbidden header that browsers did not let you set for AJAX requests, but the spec for it [changed in 2015](https://developer.mozilla.org/en-US/docs/Glossary/Forbidden_header_name$compare?locale=en-US&to=917671&from=771365) so it should be allowed now.

Maybe the maintainers could add a minimum browser versions to the documentation.

**Comment by jpfiorilla** [comment]
Hey Alec, I'm on Chrome v. 65.0.3325.181 - released about three weeks ago.

**Comment by steveruizok** [comment]
Same issue here.

**Comment by kasrak** [comment]
Should be fixed in 0.5.3: https://github.com/Airtable/airtable.js/releases/tag/v0.5.3
